### ISSUE TITLE
Mobile design questions

### ISSUE THREAD

**Issue by kiliankoe** [fsr/ese-webseite#22]
As stated most recently in #21, the design of the page is optimized for mobile devices, which does make sense.

This is how it currently looks on my iPhone:

![img_1743](https://cloud.githubusercontent.com/assets/2625584/10048218/2e55d67c-6212-11e5-98c9-85a3b554f69b.jpg)

Are there reasons for the page layout not to extend to the left and right edges as one would expect? Looks a bit awkward.

**Comment by MarauderXtreme** [comment]
As stated in #21 there are some rules that are not the default behavior. This is one of them. I think it should be obvious that this site scales on different screen sizes and that is responsivness.
I agree that this should not be the normal behavior and I will refactor this with the version that I announced in #21.
**tl;dr;**
It scales -> mobile optimized
Needs work
